src/components/ha-menu-button.ts

```diff
@@ -0,0 +1,32 @@
+import "@polymer/paper-icon-button/paper-icon-button";
+import { property, TemplateResult, LitElement, html } from "lit-element";
+import { fireEvent } from "../common/dom/fire_event";
+
+class HaMenuButton extends LitElement {
+  @property({ type: Boolean })
+  public showMenu = false;
+
+  @property({ type: Boolean })
+  public hassio = false;
+
+  protected render(): TemplateResult | void {
+    return html`
+      <paper-icon-button
+        .icon=${`${this.hassio ? "hassio" : "hass"}:menu`}
+        @click=${this._toggleMenu}
+      ></paper-icon-button>
+    `;
+  }
+
+  // We are not going to use ShadowDOM as we're rendering a single element
+  // without any CSS used.
+  protected createRenderRoot(): Element | ShadowRoot {
```

Comment:
does this gain us some performance?

Reply:
Yeah, creation of a shadow root has some overhead, because it's an isolated environment. Although I would not be able to give any numbers. Back in our Polymer 1 days, the polyfill was faster than using native ShadowDOM 😱